packages/create/src/generators/app-haunted/templates/_package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "<%= tagName %>",
+  "license": "MIT",
+  "scripts": {
+    "start": "es-dev-server --app-index index.html --node-resolve --open --watch",
+    "storybook": "start-storybook --stories 'components/*/stories/*.stories.{js,mdx}' --node-resolve --watch --open",
```

Comment:
This will need the same changes as https://github.com/open-wc/open-wc/pull/1233 (sorry bad timing :/)